cmd/snap-repair/runner.go

```diff
@@ -330,11 +330,40 @@ var (
 	maxRepairScriptSize = 24 * 1024 * 1024
 )
 
+// repairConfig is a set of configuration data that is consumed by the
+// snap-repair command. This struct is duplicated in o/c/configcore.
+type repairConfig struct {
+	// StoreOffline is true if the store is marked as offline and should not be
+	// accessed.
+	StoreOffline bool `json:"store-offline"`
+}
+
+func isStoreOffline(path string) bool {
+	f, err := os.Open(path)
+	if err != nil {
+		return false
+	}
+	defer f.Close()
+
+	var repairConfig repairConfig
+	if err := json.NewDecoder(f).Decode(&repairConfig); err != nil {
```

Comment:
(nitpick) in a followup we should probably do somehting like: `logger.Noticef("cannot read repairConfig: %v", err)` (maybe with sync.Once()) just so that any errors are not completely going into the void (with that journalctl snapd.snap-repaire will still suface them).

Reply:
And the same above in the os.Open()